templates/channels/channel_configuration.haml

```diff
@@ -510,6 +510,47 @@
       %code= ip_address
 
 
+  -elif object.channel_type == 'FCM'
+    %h4
+      -trans "To use your Firebase Cloud Messaging channel you'll have to POST to the following URLs with the parameters below."
+
+    .clearfix
+      .config-value
+        .name
+          -trans "Name:"
+        .value {{ object.name }}
+      .config-value
+        .name
+          -trans "Address/Key:"
+        .value {{ object.address }}
+      .config-value
+        .name
+          -trans "Notification title:"
+        .value {{ object.config_json.FCM_TITLE }}
+
```

Comment:
Man, I'm sorry, I'm just not getting this for some reason.

So if someone connects a Firebase channel, from their Android app, what URLs will they be hitting? Do they actually hit the "contact register" and "receive urls" themselves or does Firebase do that?

My impression is they are hitting Firebase and Firebase hits us, is that not the case? Sorry, the firebase docs aren't super clear on this call flow either..